src/kEvent.mli

```diff
@@ -163,21 +163,21 @@ val invariant : string list -> Term.t -> Certificate.t -> bool -> unit
 (** Broadcast a step cex *)
 val step_cex :
   'a InputSystem.t -> Analysis.param -> TransSys.t -> string ->
-  (StateVar.t * Model.value list) list -> unit
+  (StateVar.t * Model.value list) list -> Property.prop_kind -> unit
 
 (** Broadcast a property status *)
-val prop_status : Property.prop_status -> 'a InputSystem.t -> Analysis.param -> TransSys.t -> string -> unit
+val prop_status : Property.prop_status -> 'a InputSystem.t -> Analysis.param -> TransSys.t -> string -> Property.prop_kind -> unit
```

Comment:
I noticed you have changed a lot of the interfaces of this module to pass the kind of the property. However, in most (if not all) the cases, you are also passing the transition system and the name of the property, which is enough to retrieve the kind of the property within the function. Could you please review whether you really need to pass the new argument or you can reuse the previous interface?